packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -113,7 +114,7 @@ class InputDateTimePicker extends React.Component {
 			(!this.props.formMode && !this.props.useTime && payload.origin !== 'INPUT')
 		) {
 			this.inputRef.focus();
-			this.closePicker();
+			setTimeout(() => this.closePicker());
```

Comment:
Not sure to understand well.
The focus is done before, and it triggers `openPicker` which will not change the state because it's already open.
Then we have the closePicker.

Reply:
But it does not work in IE...

Reply:
![IE_datepicker](https://user-images.githubusercontent.com/35027619/58165635-71e8f600-7c88-11e9-9aaa-cec7f1a0a083.gif)